Sources/SpeziAccount/Resources/Localizable.xcstrings

```diff
@@ -0,0 +1,2090 @@
+{
+  "sourceLanguage" : "en",
+  "strings" : {
+    "" : {
+
+    },
+    ", " : {
+
+    },
```

Comment:
Can we check where this two translations are coming from?
Seems like there are some `Text` views that do not use the [Text(verbatim:)](https://developer.apple.com/documentation/swiftui/text/init(verbatim:)) initializer.